lib/factories/RequestHandler.ts

```diff
@@ -0,0 +1,32 @@
+import {NextFunction, Request, RequestHandler, Response} from "express";
+import {NProducer} from "sinek";
+import LoggerInterface from "../interfaces/LoggerInterface";
+import NullLogger from "../NullLogger";
+
+export function createRequestHandler(
```

Comment:
function-name/file-name does not indicate the purpose of this handler (see other, more verbose comment)